project-generator/src/main/resources/io/syndesis/project/converter/templates/pom.xml.mustache

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
The test checks for the existance of the pom.xml, not its content. So I'm not sure why the syndesis-connectors.version property within the pom.xml should affect the test. 

The dependency on the connectors must be introduced somewhere else (not via that template). 

If it is not used, could you please remove the property ? if required we can still add it then.